### ISSUE TITLE
Using small font-size differs digits from letters

### ISSUE THREAD

**Issue by EekoS** [adobe-fonts/source-sans-pro#109]
When using the font with a small font-size (13px and around it), the digits will appear smaller than letters.
For example, for the text "Hello 7432832 KB", the number 7432832 will appear shorter than the K.

This does not happen for using the font with a large font-size (40px and so).

**Comment by frankrolf** [comment]
The figures are intentionally lower than the capital letters, a design quality observed in most typefaces.  The difference is usually welcome, but it may be visible in smaller rasterization sizes when the figures’ alignment zone may fall one pixel below the caps’ alignment zone. 
If you want figures to line up exactly with capital letters, you may use the `case` feature, which activates cap-height figures.

**Comment by EekoS** [comment]
Ok, I want to try - could you elaborate on the `case ` feature? What is it? :)

**Comment by EekoS** [comment]
I see no documentation about `case` and I still think there's a bug in the font here...

**Comment by frankrolf** [comment]
This is not a bug. Figures are smaller than capital letters – which is true for almost any typeface.
See documentation about the `case` feature for here: 
http://opentypecookbook.com/common-techniques.html
https://www.typotheque.com/fonts/echo/features
http://www.preusstype.com/techdata/otf_case.php

**Comment by EekoS** [comment]
TL;DR, but how come it's different for large font-size?
How come in 40px size, the figures and letters are the same size, but in 13px size, the figures and letters size differ?

Tahoma, Arial, Verdana -- all have same size for figures and letters in all font-sizes.
BTW, Mac also "fixes" this issue and sets them on the same size. It only appears in Windows, on any browser (Chrome, FF, IE).

**Comment by frankrolf** [comment]
Some points:
- The height of figures and capital letters in Source Sans Pro are not the same, no matter in what size.
- Arial’s figures are not the same height as its capital letters. 
- Tahoma and Verdana were drawn specifically for low-resolution rasterization on computer screens of the early ’90s. That’s why many of its design decisions were taken, one of them is aligning the height of the figures with the height of the letters. Tahoma was released 24 years ago.
- The only thing you’re really talking about is rasterization. At a smaller size, there are simply not enough pixels available to fit every detail of a typeface, so sometimes there may be a difference in height not necessarily perceived at a bigger size.
- Mac doesn’t fix anything, it just uses a different rasterization method than Windows does.

At this point, we are not discussing problems in the typeface, but your habits of perception.

Being well aware of the potential issue, Paul added the OpenType `case` feature to the font, which is supported in every browser today. I would send you a link on how to use it in CSS, but I’d just get another TL;DR back – so please look for it yourself.

**Comment by EekoS** [comment]
No hard feelings I hope :) all in good spirits...
